e2e/head_test.go

```diff
@@ -0,0 +1,407 @@
+package e2e
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"gotest.tools/v3/icmd"
+)
+
+//head (without anything -> error)
+
+func TestHead(t *testing.T) {
```

Comment:
comments ;if there are any; should be just on top of the function with a single space after `//`, [`ls_test.go`](https://github.com/peak/s5cmd/blob/8355901a7be5f970a183e7d4d16d1e3b0f2ee840/e2e/ls_test.go) might help you.

https://github.com/peak/s5cmd/blob/8355901a7be5f970a183e7d4d16d1e3b0f2ee840/e2e/ls_test.go#L607-L610

if the cases/test functions names are simple and descriptive enough you may even just omit them.

edit: there're more below, this should apply to them all.

Reply:
this comment still applies to the latest changes.